---
title: "Introduction to empdata package"
output: rmarkdown::html_vignette
vignette: >
  %\VignetteIndexEntry{empdata}
  %\VignetteEngine{knitr::rmarkdown}
  %\VignetteEncoding{UTF-8}
---

<!-- # ```{r, include = FALSE} -->
<!-- # knitr::opts_chunk$set( -->
<!-- #   collapse = TRUE, -->
<!-- #   comment = "#>" -->
<!-- # ) -->
<!-- # ``` -->

## Package overview

**`empdata`** is a collection of metagenomic data originated from the [Earth Microbiome Project](https://www.earthmicrobiome.org/) (EMP). The package contains a subset of 975 samples with 150bp-length sequences. 

Full documentation regarding the EMP dataset can be found in the related article *[A communal catalogue reveals Earth’s multiscale microbial diversity](https://doi.org/10.1038/nature24621)*.

## Sample collection

Samples were gathered for microbiome analysis. 
Each sample was collected fresh and, where possible, immediately frozen in liquid nitrogen and stored at –80 °C. 
Bulk samples (for example, soil, sediment, faeces) and fractionated bulk samples (for example, sponge coral surface tissue, centrifuged turbid water) were taken using microcentrifuge tubes. 
Swabs (BD SWUBE dual cotton swabs or similar) were used for biofilm or surface samples. 
Filters (Sterivex cartridges, 0.2 μm, Millipore) were used for water samples. 
Samples were sent to laboratories in the United States for DNA extraction and sequencing: 
water samples to Argonne National Laboratory, 
soil samples to Lawrence Berkeley National Laboratory (pre-2014) or Pacific Northwest National Laboratory (2014 onward), 
and faecal and other samples to the University of Colorado Boulder (pre-2015) or the University of California San Diego (2015 onward).

## Metadata curation

Metadata were collected in compliance with MIMARKS, [EBI](https://www.ebi.ac.uk/ena), and [Qiita](http://qiita.microbio.me) standards, as described in the [EMP Metadata Guide](http://www.earthmicrobiome.org/protocols-and-standards/metadata-guide). 

### The EMP Ontology

The EMP Ontology (EMPO) for microbial environments was devised to facilitate the present analysis while preserving interoperability and it was coordinated by the ENVO team.
It contains annotations from ENVO, UBERON (metazoan anatomy), PO (plant ontology), [FAO](http://purl.obolibrary.org/obo/fao.owl)(fungal anatomy ontology), and OMP (ontology of microbial phenotypes)
and it consists of three-level categories:

- *empo_1*: free-living or host-associated lifestyles
- *empo_2*: saline or non-saline (if free-living) or animal or plant (if host-associated) 
- *empo_3*: microbial environments

Full descriptions of the EMP Ontology categories can be found at http://www.earthmicrobiome.org/protocols-and-standards/empo. 
The W3C Web Ontology Language (OWL) document is available at http://purl.obolibrary.org/obo/envo/subsets/envoEmpo.owl. 

## DNA extraction, amplicon PCR, sequencing, and sequence pre-processing

DNA extraction and 16S rRNA amplicon sequencing were done based on the [EMP standard protocols](http://www.earthmicrobiome.org/protocols-and-standards/16s).

## Tag sequence and subsets

The overall EMP dataset is provided throughout the [FTP site](ftp://ftp.microbio.me/emp/release1/) (release1) and it contains a total of 97 studies and 27,742 samples.

Sequence data were error-filtered and trimmed to three different sequence lengths:

- The length of the shortest sequencing run (90 bp)
- Sequence data trimmed to 100 bp (greater sequence resolution but fewer samples)
- Sequence data trimmed to 150 bp (greater sequence resolution but fewer samples)

Additionally, samples were randomly picked to give equal (as possible) representation across environments (EMPO level 3) and studies within those environments. 
The subsets contain 10,000, 5,000, and 2,000 samples (nested subsets). 

## Package subset

For the package development, the subset of 2,000 samples with 150-bp length was chosen. 
The related BIOM file was downloaded from the [FTP site](ftp://ftp.microbio.me/emp/release1/otu_tables/deblur/) (*emp_deblur_150bp.subset_2k.rare_5000.biom*).
Sample metadata were also downloaded from the [FTP server](ftp://ftp.microbio.me/emp/release1/mapping_files/) (*emp_qiime_mapping_subset_2k.tsv*). 

The metadata file was additionally filtered to contain only the samples from the downloaded BIOM file. 
The resulting dataset contains 975 samples with sequences of 150bp length.
