<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Sample metadata table — sample_metadata • empdata</title>


<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script>

<!-- bootstrap-toc -->
<link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>




<meta property="og:title" content="Sample metadata table — sample_metadata" />
<meta property="og:description" content="This is a meta-data/mapping file containing related information about samples originated from the Earth Microbiome Project." />


<meta name="robots" content="noindex">

<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body data-spy="scroll" data-target="#toc">
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">empdata</a>
        <span class="version label label-danger" data-toggle="tooltip" data-placement="bottom" title="In-development version">0.0.1.2</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../articles/empdata.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/nikopech/empdata">
    <span class="fab fa-github"></span>
     
    GitHub
  </a>
</li>
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Sample metadata table</h1>
    
    <div class="hidden name"><code>sample_metadata.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>This is a meta-data/mapping file containing related information about samples originated from the Earth Microbiome Project.</p>
    </div>

    <pre class="usage"><span class='va'>sample_metadata</span></pre>


    <h2 class="hasAnchor" id="format"><a class="anchor" href="#format"></a>Format</h2>

    <p>Data frame with columns</p><dl>
<dt>#SampleID</dt><dd><p>unique sample identifier (leading hashtag is required by Qiime)</p></dd>
<dt>BarcodeSequence</dt><dd><p>barcode sequence</p></dd>
<dt>LinkerPrimerSequence</dt><dd><p>linker primer sequence</p></dd>
<dt>Description</dt><dd><p>sample description</p></dd>
<dt>host_subject_id</dt><dd><p>unique host or subject (can have multiple samples per host_subject_id)</p></dd>
<dt>study_id</dt><dd><p>study identifier</p></dd>
<dt>title</dt><dd><p>study title</p></dd>
<dt>principal_investigator</dt><dd><p>PI of the study</p></dd>
<dt>doi</dt><dd><p>Digital Object Identifier (DOI) or PubMed ID of primary publication</p></dd>
<dt>ebi_accession</dt><dd><p>European Bioinformatics Institute (EBI) European Nucleotide Archive (ENA) accession number if submitted</p></dd>
<dt>target_gene</dt><dd><p>name of gene amplified (e.g., 16S rRNA)</p></dd>
<dt>target_subfragment</dt><dd><p>name of subfragment of gene amplified (e.g., V4)</p></dd>
<dt>pcr_primers</dt><dd><p>amplicon primer sequences used</p></dd>
<dt>illumina_technology</dt><dd><p>model of Illumina sequencer</p></dd>
<dt>extraction_center</dt><dd><p>where the sample was physically extracted</p></dd>
<dt>run_center</dt><dd><p>where the sample was physically sequenced (CCME=U Colorado Boulder, ANL=Argonne Natl Lab, UCSDMI=UC San Diego, CGS-GL=Wash U, UCD=U Colorado Denver)</p></dd>
<dt>run_date</dt><dd><p>date the sample was physically sequenced</p></dd>
<dt>read_length_bp</dt><dd><p>median read length in bp across study after quality filtering</p></dd>
<dt>sequences_split_libraries</dt><dd><p>number of sequences after demultiplexing with split_libraries_fastq.py</p></dd>
<dt>observations_closed_ref_greengenes</dt><dd><p>number of observations in closed-reference Greengenes table</p></dd>
<dt>observations_closed_ref_silva</dt><dd><p>number of observations in closed-reference Silva table</p></dd>
<dt>observations_open_ref_greengenes</dt><dd><p>number of observations in open-reference Greengenes table</p></dd>
<dt>observations_deblur_90bp</dt><dd><p>number of observations in 90-bp Deblur table</p></dd>
<dt>observations_deblur_100bp</dt><dd><p>number of observations in 100-bp Deblur table</p></dd>
<dt>observations_deblur_150bp</dt><dd><p>number of observations in 150-bp Deblur table</p></dd>
<dt>emp_release1</dt><dd><p>samples with &gt;=1 sequences (split libraries) per sample</p></dd>
<dt>qc_filtered</dt><dd><p>samples with &gt;=1000 observations in CR-GG &amp; CR-Silva &amp; OR-GG &amp; Deblur-90 but excluding controls (all subsets are in this set)</p></dd>
<dt>subset_10k</dt><dd><p>10000 samples with &gt;=10000 CR-GG &amp; CR-Silva &amp; OR-GG and &gt;=5000 Deblur-90, randomly selected and evenly distributed across <code>empo_3</code> categories and then across studies</p></dd>
<dt>subset_5k</dt><dd><p>5000 samples with &gt;=10000 CR-GG &amp; CR-Silva &amp; OR-GG and &gt;=5000 Deblur-90, randomly selected and evenly distributed across <code>empo_3</code> categories and then across studies</p></dd>
<dt>subset_2k</dt><dd><p>2000 samples with &gt;=10000 CR-GG &amp; CR-Silva &amp; OR-GG and &gt;=5000 Deblur-90, randomly selected and evenly distributed across <code>empo_3</code> categories and then across studies</p></dd>
<dt>sample_taxid</dt><dd><p>sample NCBI taxonomy ID</p></dd>
<dt>sample_scientific_name</dt><dd><p>sample NCBI scientific name looked up from taxonomy ID</p></dd>
<dt>host_taxid</dt><dd><p>host NCBI taxonomy ID</p></dd>
<dt>host_common_name_provided</dt><dd><p>host common name provided in the original mapping file</p></dd>
<dt>host_common_name</dt><dd><p>host NCBI common name looked up from taxonomy ID</p></dd>
<dt>host_scientific_name</dt><dd><p>host NCBI scientific name looked up from taxonomy ID</p></dd>
<dt>host_superkingdom</dt><dd><p>host superkingdom looked up from taxonomy ID</p></dd>
<dt>host_kingdom</dt><dd><p>host kingdom looked up from taxonomy ID</p></dd>
<dt>host_phylum</dt><dd><p>host phylum looked up from taxonomy ID</p></dd>
<dt>host_class</dt><dd><p>host class looked up from taxonomy ID</p></dd>
<dt>host_order</dt><dd><p>host order looked up from taxonomy ID</p></dd>
<dt>host_family</dt><dd><p>host family looked up from taxonomy ID</p></dd>
<dt>host_genus</dt><dd><p>host genus looked up from taxonomy ID</p></dd>
<dt>host_species</dt><dd><p>host species looked up from taxonomy ID</p></dd>
<dt>collection_timestamp</dt><dd><p>date and time when sample was collected</p></dd>
<dt>country</dt><dd><p>country where sample was collected</p></dd>
<dt>latitude_deg</dt><dd><p>latitude in degrees</p></dd>
<dt>longitude_deg</dt><dd><p>longitude in degrees</p></dd>
<dt>depth_m</dt><dd><p>depth in meters of sample below surface (earth surface if soil, sea/lake bottom if sediment, lake surface if lake, sea level if marine)</p></dd>
<dt>altitude_m</dt><dd><p>height above surface, usually zero unless the mouse is levitating (either depth or altitude can have a non-zero value but not both)</p></dd>
<dt>elevation_m</dt><dd><p>height above sea level in meters (from georeferencing tool)</p></dd>
<dt>env_biome</dt><dd><p>ENVO biome</p></dd>
<dt>env_feature</dt><dd><p>ENVO feature</p></dd>
<dt>env_material</dt><dd><p>ENVO material</p></dd>
<dt>envo_biome_0</dt><dd><p>level 0 ENVO biome looked up from given ENVO biome</p></dd>
<dt>envo_biome_1</dt><dd><p>level 1 ENVO biome looked up from given ENVO biome</p></dd>
<dt>envo_biome_2</dt><dd><p>level 2 ENVO biome looked up from given ENVO biome</p></dd>
<dt>envo_biome_3</dt><dd><p>level 3 ENVO biome looked up from given ENVO biome</p></dd>
<dt>envo_biome_4</dt><dd><p>level 4 ENVO biome looked up from given ENVO biome</p></dd>
<dt>envo_biome_5</dt><dd><p>level 5 ENVO biome looked up from given ENVO biome</p></dd>
<dt>empo_0</dt><dd><p>level 0 EMPO habitat from EMPO ontology</p></dd>
<dt>empo_1</dt><dd><p>level 1 EMPO habitat from EMPO ontology</p></dd>
<dt>empo_2</dt><dd><p>level 2 EMPO habitat from EMPO ontology</p></dd>
<dt>empo_3</dt><dd><p>level 3 EMPO habitat from EMPO ontology</p></dd>
<dt>adiv_observed_otus</dt><dd><p>observed tag sequences of 90-bp Deblur table rarefied to 5000 sequences per sample</p></dd>
<dt>adiv_chao1</dt><dd><p>Chao1 index of 90-bp Deblur table rarefied to 5000 sequences per sample</p></dd>
<dt>adiv_shannon</dt><dd><p>Shannon index of 90-bp Deblur table rarefied to 5000 sequences per sample</p></dd>
<dt>adiv_faith_pd</dt><dd><p>Faith's phylogenetic diversity of 90-bp Deblur table rarefied to 5000 sequences per sample</p></dd>
<dt>temperature_deg_c</dt><dd><p>ambient temperature in degrees Celcius</p></dd>
<dt>ph</dt><dd><p>pH value</p></dd>
<dt>salinity_psu</dt><dd><p>salinity in practical salinity units (PSU)</p></dd>
<dt>oxygen_mg_per_l</dt><dd><p>oxygen concentration in mg/L</p></dd>
<dt>phosphate_umol_per_l</dt><dd><p>phosphate concentration in µmol/L</p></dd>
<dt>ammonium_umol_per_l</dt><dd><p>ammonium concentration in µmol/L</p></dd>
<dt>nitrate_umol_per_l</dt><dd><p>nitrate concentration in µmol/L</p></dd>
<dt>sulfate_umol_per_l</dt><dd><p>sulfate concentration in µmol/L</p></dd>

</dl>

    <h2 class="hasAnchor" id="source"><a class="anchor" href="#source"></a>Source</h2>

    <p><a href='https://earthmicrobiome.org/'>https://earthmicrobiome.org/</a></p>

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'>  <span class='fu'><a href='https://rdrr.io/r/utils/head.html'>head</a></span><span class='op'>(</span><span class='va'>sample_metadata</span><span class='op'>)</span>
</div><div class='output co'>#&gt;   #SampleID BarcodeSequence LinkerPrimerSequence
#&gt; 1    662.M1    CGAGTCACGATT  GTGCCAGCMGCCGCGGTAA
#&gt; 2    662.M2    GCCATAGTGTGT  GTGCCAGCMGCCGCGGTAA
#&gt; 3    662.M3    GTAGACATGTGT  GTGCCAGCMGCCGCGGTAA
#&gt; 4    662.M4    TGTGGCTCGTGT  GTGCCAGCMGCCGCGGTAA
#&gt; 5   662.M5a    TCTGATCGAGGT  GTGCCAGCMGCCGCGGTAA
#&gt; 6   662.M5b    AGAGAGACAGGT  GTGCCAGCMGCCGCGGTAA
#&gt;                             Description host_subject_id study_id
#&gt; 1  biofilm brushed from crucible lid M1              M1      662
#&gt; 2  biofilm brushed from crucible lid M2              M2      662
#&gt; 3  biofilm brushed from crucible lid M3              M3      662
#&gt; 4  biofilm brushed from crucible lid M4              M4      662
#&gt; 5 biofilm brushed from crucible lid M5a             M5a      662
#&gt; 6 biofilm brushed from crucible lid M5b             M5b      662
#&gt;                                                                            title
#&gt; 1 The role of macrobiota in structuring microbial communities along rocky shores
#&gt; 2 The role of macrobiota in structuring microbial communities along rocky shores
#&gt; 3 The role of macrobiota in structuring microbial communities along rocky shores
#&gt; 4 The role of macrobiota in structuring microbial communities along rocky shores
#&gt; 5 The role of macrobiota in structuring microbial communities along rocky shores
#&gt; 6 The role of macrobiota in structuring microbial communities along rocky shores
#&gt;   principal_investigator               doi ebi_accession target_gene
#&gt; 1      Catherine Pfister 10.7717/peerj.631     ERP020507    16S rRNA
#&gt; 2      Catherine Pfister 10.7717/peerj.631     ERP020507    16S rRNA
#&gt; 3      Catherine Pfister 10.7717/peerj.631     ERP020507    16S rRNA
#&gt; 4      Catherine Pfister 10.7717/peerj.631     ERP020507    16S rRNA
#&gt; 5      Catherine Pfister 10.7717/peerj.631     ERP020507    16S rRNA
#&gt; 6      Catherine Pfister 10.7717/peerj.631     ERP020507    16S rRNA
#&gt;   target_subfragment                                       pcr_primers
#&gt; 1                 V4 FWD:GTGCCAGCMGCCGCGGTAA; REV:GGACTACHVGGGTWTCTAAT
#&gt; 2                 V4 FWD:GTGCCAGCMGCCGCGGTAA; REV:GGACTACHVGGGTWTCTAAT
#&gt; 3                 V4 FWD:GTGCCAGCMGCCGCGGTAA; REV:GGACTACHVGGGTWTCTAAT
#&gt; 4                 V4 FWD:GTGCCAGCMGCCGCGGTAA; REV:GGACTACHVGGGTWTCTAAT
#&gt; 5                 V4 FWD:GTGCCAGCMGCCGCGGTAA; REV:GGACTACHVGGGTWTCTAAT
#&gt; 6                 V4 FWD:GTGCCAGCMGCCGCGGTAA; REV:GGACTACHVGGGTWTCTAAT
#&gt;   illumina_technology extraction_center run_center run_date read_length_bp
#&gt; 1               HiSeq       ANL-Chicago        ANL   6/2/11            140
#&gt; 2               HiSeq       ANL-Chicago        ANL   6/2/11            140
#&gt; 3               HiSeq       ANL-Chicago        ANL   6/2/11            140
#&gt; 4               HiSeq       ANL-Chicago        ANL   6/2/11            140
#&gt; 5               HiSeq       ANL-Chicago        ANL   6/2/11            140
#&gt; 6               HiSeq       ANL-Chicago        ANL   6/2/11            140
#&gt;   sequences_split_libraries observations_closed_ref_greengenes
#&gt; 1                    298351                             275292
#&gt; 2                    281588                             219846
#&gt; 3                    187735                             165334
#&gt; 4                    271744                             221436
#&gt; 5                    230631                             140483
#&gt; 6                    217961                             134035
#&gt;   observations_closed_ref_silva observations_open_ref_greengenes
#&gt; 1                        280849                           297717
#&gt; 2                        230554                           280463
#&gt; 3                        168100                           186813
#&gt; 4                        231443                           270818
#&gt; 5                        167126                           229193
#&gt; 6                        169479                           217037
#&gt;   observations_deblur_90bp observations_deblur_100bp observations_deblur_150bp
#&gt; 1                   201116                    194458                     96458
#&gt; 2                   177503                    174092                     90406
#&gt; 3                   119184                    118020                     64784
#&gt; 4                   181115                    170197                     86880
#&gt; 5                   142987                    138395                     63409
#&gt; 6                   139000                    133015                     68383
#&gt;   emp_release1 qc_filtered subset_10k subset_5k subset_2k sample_taxid
#&gt; 1         TRUE        TRUE       TRUE      TRUE      TRUE       718308
#&gt; 2         TRUE        TRUE       TRUE      TRUE      TRUE       718308
#&gt; 3         TRUE        TRUE       TRUE      TRUE      TRUE       718308
#&gt; 4         TRUE        TRUE       TRUE      TRUE      TRUE       718308
#&gt; 5         TRUE        TRUE       TRUE      TRUE      TRUE       718308
#&gt; 6         TRUE        TRUE       TRUE      TRUE      TRUE       718308
#&gt;   sample_scientific_name host_taxid host_common_name_provided host_common_name
#&gt; 1     biofilm metagenome         NA                                           
#&gt; 2     biofilm metagenome         NA                                           
#&gt; 3     biofilm metagenome         NA                                           
#&gt; 4     biofilm metagenome         NA                                           
#&gt; 5     biofilm metagenome         NA                                           
#&gt; 6     biofilm metagenome         NA                                           
#&gt;   host_scientific_name host_superkingdom host_kingdom host_phylum host_class
#&gt; 1                                                                           
#&gt; 2                                                                           
#&gt; 3                                                                           
#&gt; 4                                                                           
#&gt; 5                                                                           
#&gt; 6                                                                           
#&gt;   host_order host_family host_genus host_species collection_timestamp
#&gt; 1                                                 2009-08-06 00:00:00
#&gt; 2                                                 2009-08-06 00:00:00
#&gt; 3                                                 2009-08-06 00:00:00
#&gt; 4                                                 2009-08-06 00:00:00
#&gt; 5                                                 2009-08-06 00:00:00
#&gt; 6                                                 2009-08-06 00:00:00
#&gt;                        country latitude_deg longitude_deg depth_m altitude_m
#&gt; 1 GAZ:United States of America        48.38       -124.73       0          0
#&gt; 2 GAZ:United States of America        48.38       -124.73       0          0
#&gt; 3 GAZ:United States of America        48.38       -124.73       0          0
#&gt; 4 GAZ:United States of America        48.38       -124.73       0          0
#&gt; 5 GAZ:United States of America        48.38       -124.73       0          0
#&gt; 6 GAZ:United States of America        48.38       -124.73       0          0
#&gt;   elevation_m    env_biome     env_feature env_material envo_biome_0
#&gt; 1       24.73 marine biome intertidal zone      biofilm        biome
#&gt; 2       24.73 marine biome intertidal zone      biofilm        biome
#&gt; 3       24.73 marine biome intertidal zone      biofilm        biome
#&gt; 4       24.73 marine biome intertidal zone      biofilm        biome
#&gt; 5       24.73 marine biome intertidal zone      biofilm        biome
#&gt; 6       24.73 marine biome intertidal zone      biofilm        biome
#&gt;    envo_biome_1 envo_biome_2 envo_biome_3 envo_biome_4 envo_biome_5     empo_0
#&gt; 1 aquatic biome marine biome                                        EMP sample
#&gt; 2 aquatic biome marine biome                                        EMP sample
#&gt; 3 aquatic biome marine biome                                        EMP sample
#&gt; 4 aquatic biome marine biome                                        EMP sample
#&gt; 5 aquatic biome marine biome                                        EMP sample
#&gt; 6 aquatic biome marine biome                                        EMP sample
#&gt;        empo_1 empo_2           empo_3 adiv_observed_otus adiv_chao1
#&gt; 1 Free-living Saline Surface (saline)                 95   116.0833
#&gt; 2 Free-living Saline Surface (saline)                187   263.5556
#&gt; 3 Free-living Saline Surface (saline)                190   230.5294
#&gt; 4 Free-living Saline Surface (saline)                181   261.4545
#&gt; 5 Free-living Saline Surface (saline)                195   283.6667
#&gt; 6 Free-living Saline Surface (saline)                181   202.3030
#&gt;   adiv_shannon adiv_faith_pd temperature_deg_c ph salinity_psu oxygen_mg_per_l
#&gt; 1     3.627853      11.74937                NA NA           NA              NA
#&gt; 2     3.898378      22.86418                NA NA           NA              NA
#&gt; 3     4.520886      22.38092                NA NA           NA              NA
#&gt; 4     4.220427      22.08485                NA NA           NA              NA
#&gt; 5     5.141952      25.27403                NA NA           NA              NA
#&gt; 6     4.554962      22.64086                NA NA           NA              NA
#&gt;   phosphate_umol_per_l ammonium_umol_per_l nitrate_umol_per_l
#&gt; 1                   NA                  NA                 NA
#&gt; 2                   NA                  NA                 NA
#&gt; 3                   NA                  NA                 NA
#&gt; 4                   NA                  NA                 NA
#&gt; 5                   NA                  NA                 NA
#&gt; 6                   NA                  NA                 NA
#&gt;   sulfate_umol_per_l
#&gt; 1                 NA
#&gt; 2                 NA
#&gt; 3                 NA
#&gt; 4                 NA
#&gt; 5                 NA
#&gt; 6                 NA</div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top">
      <h2 data-toc-skip>Contents</h2>
    </nav>
  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by <a href='https://biodataanalysisgroup.github.io/team/'>Nikolaos Pechlivanis</a>.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


